game/src/ecs/systems/HealthSystem.java

```diff
@@ -0,0 +1,64 @@
+package ecs.systems;
+
+import ecs.components.AnimationComponent;
+import ecs.components.HealthComponent;
+import ecs.components.MissingComponentException;
+import ecs.damage.Damage;
+import ecs.entities.Entity;
+import java.util.List;
+import mydungeon.ECS;
+
+/**
+ * The HealthSystem offsets the damage to be done to all entities with the HealthComponent. Triggers
+ * the death of an entity when the hitpoints have fallen below 0.
+ */
+public class HealthSystem extends ECS_System {
+    @Override
+    public void update() {
```

Comment:
Örgs. Gab es da nicht eine bessere Lösung?

Reply:
Für was genau?

Reply:
Für dieses wahnsinnig eingerückte Konstrukt. Wer soll denn sowas lesen?

Reply:
Okay. Ich dachte du willst das `@Override` anmeckern.
Du hattest ja bereits modernes Java "vorgeshclagen" #270
Ist aber dann ein Ding für sich. Hier ist das noch vor diesem Wissen entstanden.

Reply:
Hmmm. Aber jetzt weisst Du das und könntest das doch gleich einbauen? Scheint mir einfacher als das erstmal flink einkippen und dann ein Ticket zu haben, um es nachträglich zu flicken?

Reply:
Ja. Aber ich merk bei der Umsetzung grade noch Schwierigkeiten (vermutlich aus Unerfahrenheit im Thema) und würde das gerne "in ruhe" machen.
Dieses PR wird noch für andere Sachen gebraucht und daher lieber auslagern